charts/gardener-extension-admission-gcp/values.yaml

```diff
@@ -7,6 +7,14 @@ global:
     pullPolicy: IfNotPresent
   replicaCount: 1
   resources: {}
+  vpa:
+    enabled: true
+  # resourcePolicy:
+  #   minAllowed:
+  #     cpu: 100m
+  #     memory: 128Mi
```

Comment:
Let's have some minAllowed by default. I remember we had issues with pods scaled down so much that then the VPA was not able to scale-up them anymore.

Reply:
Which values do you propose? I can add something here, but I'm not sure whether it's too helpful. It highly depends on the landscape's size, so any default here will only work for small/"hello world" landscapes, I believe.

Reply:
I believe those are already good, they just need to be uncommented.

Reply:
Hm, still not sure. This will unnecessarily increase the values on small landscapes. For example, this is the current memory usage for a landscape with 180 shoots, 22 seeds, and ~1k secrets.

```
gardener-extension-admission-gcp-7dd46c9ddf-fgfbl         1m           27Mi
gardener-extension-admission-gcp-7dd46c9ddf-kcqkm         1m           26Mi
gardener-extension-admission-gcp-7dd46c9ddf-p4697         6m           26Mi
```

We can have rather small "min allowed" values (`10m`/`30Mi`, for example), they should be fine for most landscapes. For larger landscapes, the values need to be overridden anyways.
WDYT?

Reply:
This is running just once per landscape, so it will not save so much resources. While, the lower the values, the higher the risk the VPA to stuck and to not up-scale the resources. I would set it to at least 50m CPU and 64MiB of memory.